firmware/user/modes/mode_color_movement.c

```diff
@@ -76,7 +76,7 @@
 #define MS_TO_US_FACTOR 1000
 #define S_TO_MS_FACTOR 1000
 #define TITLE_VISIBLE_DURATION 4000000
-#define BPM_SAMPLE_TIME 12000 // in ms
+#define BPM_SAMPLE_TIME 4000 // 12000 // in ms
```

Comment:
@bbkiwi is there any visual difference when cutting this number by two thirds?

Reply:
Yes graphs for bpm submodes not as smooth and leds more variable. Your PR is better and forgot this one. (Sad to see CC vanish)

Reply:
CC can always come back, that's not tremendously hard to do. It was hard to justify keeping all that code around when it did nothing this year.